src/storybook/stand-alone-documentaion/typography/typography.stories.mdx

```diff
@@ -0,0 +1,50 @@
+import { useState, useCallback } from "react";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { Link, RelatedComponents } from "../../components";
+import { TextStyles } from "./text-styles/text-styles";
+import { TextColors } from "./text-colors/text-colors";
+import { COLORS } from "../../components/related-components/component-description-map";
+import {typographyExample1, typographyExample2} from "./assets";
+import "./typography.scss";
+
+<Meta
+  title="Foundations/Typography"
+/>
+
+<!--- Component documentation -->
+
+# Typography
+- [Overview](#overview)
+- [Text styles](#text-styles)
+- [Text colors](#text-colors)
+- [Usage and examples](#usage-and-examples)
+- [Feedback](#feedback)
+
+## Overview
+Like in other ui in mondays interface, typography works by principle of accessibility before aesthetics.
+Therefore the text should be readable and help the user understand what’s important by well contrasted size and colors hierarchy.
+
+## Text styles
+<TextStyles />
+
+## Text colors
+<TextColors />
+
+## Usage and examples
+<UsageGuidelines guidelines={[
+  "Use fonts bigger than 14 pixel only.",
+  <>Don’t underline words, use bold text.</>,
+  "Don't use two different sizes of text in the same line."
+]}/>
+
+<Tip title="What font to use?">
+  We are using Roboto from Google Fonts as default for our design system. Click
+  <Link size={Link.sizes.SMALL} href="https://fonts.google.com/specimen/Roboto" withoutSpacing>here</Link>
+  to download.
+</Tip>
+
+<img src={typographyExample1} alt="" className="monday-storybook-typography_first-image"/>
```

Comment:
(see image) UI H6 - we don't use h6 for that, it is not a heading

Reply:
I will ask for fixed image from evigany